pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -494,6 +515,165 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
 }
 ```
 
+**2. Get details for Transactions - Version 2.0**
+
+Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api-docs/?api=reporting#/Transaction/transactions) to go through API spec to get started.
+
+| Method | URI  |
+| :--- | :--- |
+| POST | https://{domain}/api/account/{accountId}/app/{app}/transactions/
+
+**Path Parameters**
+
+| Name  | Description | Required | Value/Example |
+| :--- | :--- | :--- | :--- |
+| domain   | domain | Yes | va.cx-reporting.liveperson.net or lo.cx-reporting.liveperson.net or sy.cx-reporting.liveperson.net |
+| accountId | LivePerson site ID | Yes | 12345678 |
+| app | App name | Yes | "prmsg" or "c2m" |
+
+**Query Parameters**
+
+| Name  | Description | Required | Value/Example |
+| :--- | :--- | :--- | :--- |
+| attemptedStartTime | Starting time (epoch milliseconds) of attempted events | Yes | 1602007811000 |
+| attemptedEndTime | Ending time (epoch milliseconds) of attempted events | Yes | 1602008344000 |
+| v | Transaction API version | Yes | 2.0 |
+
+**Request Headers**
+
+| Header | Description | Value/Example |
+| :--- | :--- | :--- |
+| Content-Type | Used to indicate the media type of the resource | application/json |
+| Authorization | [OAuth 2.0](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt) or [OAuth 1.0](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html) (Section 8) or LE Bearer token | |
+
+**Request Body**
+
+| Key | Description | Values/Examples |
+| :--- | :--- | :--- |
+| offset | The offset specifies from which record to retrieve the data | Default = 0 |
```

Comment:
mention that it is ordered by transaction id

Reply:
I dont think this is required. Ultimately user will get data sorted by transaction hour.